migration/v0.38/prepare/30-check-upcloud-s3-version.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Can you also add this to the [manual steps](https://github.com/elastisys/compliantkubernetes-apps/tree/main/migration/v0.38#prepare-upgrade---non-disruptive) with a disclaimer that its only for Upcloud.

Reply:
Right, I forgot about that, thanks!